fe2-android/app/src/test/ui/robolectric/com/github/dedis/popstellar/ui/home/LaoCreateFragmentTest.java

```diff
@@ -87,11 +91,18 @@ public void setup() {
   }
 
   @Test
-  public final void uiElementsAreDisplayed() {
+  public final void uiElementsAreCorrectlyDisplayed() {
     laoNameEntry().check(matches(isDisplayed()));
-    cancelButtonLaunch().check(matches(isDisplayed()));
+    serverNameEntry().check(matches(isDisplayed()));
+    clearButtonLaunch().check(matches(isDisplayed()));
     confirmButtonLaunch().check(matches(isDisplayed()));
-    addWitnessButton().check(matches(isDisplayed()));
+    witnessingSwitch().check(matches(isDisplayed()));
+  }
+
+  @Test
+  public void uiElementsAreCorrectlyHidden() {
```

Comment:
You could add a comment here or change the test name to clarify that this is the state where the witnessing switch is off.